platform/microprofile/src/main/java/org/trellisldp/microprofile/TrellisServiceSupplier.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.trellisldp.microprofile;
+
+import static org.eclipse.microprofile.config.ConfigProvider.getConfig;
+import static org.trellisldp.triplestore.TriplestoreResourceService.CONFIG_TRIPLESTORE_RDF_LOCATION;
+import static org.trellisldp.triplestore.TriplestoreResourceService.buildRDFConnection;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Produces;
+
+import org.apache.jena.rdfconnection.RDFConnection;
+import org.trellisldp.api.*;
+
+/**
+ * A managed bean that generates an RDF connection for the triplestore resource service.
+ */
+@ApplicationScoped
+public class TrellisServiceSupplier {
```

Comment:
Does this belong in the main µprofile artifact, or in something specific to the triplestore backend? If I want to reuse the main artifact, am I going to have to work around this in order to use my `EventService` of choice?

Reply:
This class is definitely specific to the triplestore back-end with a particular selection of service implementations. What I was trying to achieve here was a specific microprofile-based deployment that could be used as an example for other µP applications. The `trellis-app` is really intended to be the shared layer that is built upon for µP, dropwizard, spring, etc.

Reply:
Ok, that's a good strategy, but in that case, I think we should be more upfront about this artifact being an _example_ and not a _reusable part_. Maybe a name change `microprofile`->`triplestore`? Given that we expect to deprecate Dropwizard eventually, do we even need to call out µmicroprofile explicitly?

Reply:
That's a good point. I'd want to be able to distinguish between this and the `./components/triplestore` project (and also the `trellis-triplestore` artifact). With the Dropwizard-based code, I've used `app-triplestore` and similar as a naming convention. For now, how about `mp-triplestore` or `app-mp-triplestore`? Or maybe some other name?

Reply:
I like `app-mp-triplestore` because it has all the important facts. Let's roll with that!